spec/ca_testing/extensions/capybara/node/element_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+describe Capybara::Node::Element do
+  subject(:capybara_element) { described_class.new(:_session, :_base, :_query_scope, :_query) }
+  let(:selenium_element) { double(rect: rect) }
+  let(:rect) { OpenStruct.new(x: 8, y: 29, width: 185, height: 21) }
```

Comment:
So originally I was going to say because it's needed. However after realising the `rect` generated by selenium it itself a struct, we can use **both** fetch and accessor methods. So I've altered the extension methods to use accessor methods and consequently edited this to be an instance double so it's a bit better.